### ISSUE TITLE
Broken due to removed dependency

### ISSUE THREAD

**Issue by erquhart** [kisenka/svg-sprite-loader#205]
**Do you want to request a feature, report a bug or ask a question?**

bug

**What is the current behavior?**

loader doesn't work due to removal of `posthtml-svg-mode` from npm

**What is the expected behavior?**

loader should work

Related issue in `svg-baker` repo: https://github.com/kisenka/svg-baker/issues/17

**Comment by kisenka** [comment]
I didn't remove it from NPM, just change posthtml-svg-mode readme about renaming it to postsvg. Could you please provide an error or warning which happens when you install svg-sprite-loader.

**Comment by kisenka** [comment]
@erquhart btw which package manager is used?

**Comment by erquhart** [comment]
@kisenka the module contains a readme, a package.json, and nothing else, so you'll get a "can't find module" error. Using yarn/webpack, I get this:

```
Module build failed: Error: Cannot find module 'posthtml-svg-mode'
```

You didn't update the dependency in `svg-baker`: https://github.com/kisenka/svg-baker/blob/master/packages/svg-baker/package.json#L26

**Comment by aga5tya** [comment]
@kisenka Same here with yarn after latest version bump.

**Comment by erquhart** [comment]
@kisenka if you're deprecating, the 1.0.1 patch should simply include a notice that the module is being deprecated, and the next major release (2.0) would bring removal. As it stands, this isn't a deprecation, but a breaking change.

**Comment by kisenka** [comment]
@erquhart seems like it was an error while publishing, I've published posthtml-svg-mode @1.0.2 and recheck package contents.

**Comment by erquhart** [comment]
Awesome, thank you!
